sources/engine/Stride.Graphics/Direct3D12/GraphicsDevice.Direct3D12.Pools.cs

```diff
@@ -0,0 +1,247 @@
+// Copyright (c) .NET Foundation and Contributors (https://dotnetfoundation.org/ & https://stride3d.net) and Silicon Studio Corp. (https://www.siliconstudio.co.jp)
+// Distributed under the MIT license. See the LICENSE.md file in the project root for more information.
```

Comment:
If it's a new file, the header shouldn't include SiliconStudio. Only legacy files that were created during the Xenko's time have that.

```csharp
// Copyright (c) .NET Foundation and Contributors (https://dotnetfoundation.org/ & https://stride3d.net)
// Distributed under the MIT license. See the LICENSE.md file in the project root for more information.
```